client/src/components/Draftail/CommentableEditor/CommentableEditor.tsx

```diff
@@ -887,10 +887,11 @@ function CommentableEditor({
                 background = focusedHighlight;
                 return {
                   'background-color': background,
-                  'color': standardHighlight,
+                  'font-weight': '700',
```

Comment:
Just a heads up - might be good to incorporate this other fix I have prepared. 

https://github.com/wagtail/wagtail/pull/9082

Reply:
Ooh nice I've added those changes to this PR now too 🙂